### PR TITLE
feat: dockerify + kubernetesable

### PR DESCRIPTION
**Pull request by bodinsamuel**
Just a test for the moment,
I'm testing the script in a machine in our private google cloud kubernetes cluster (1cpu/2GO ram) to see how it goes.

### CODE REVIEW

**Comment by bodinsamuel** [comment]
JSON stream error happened at the exact same second as the prod one 👌 
Still don't know why but at least it's consistant

File: kubernetes.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            - yarn
+            - start
+          resources:
+            requests:
+              cpu: '1'
+              memory: '2000Mi'
+
+          env:
+            - name: appId
+              value: 'KECOEVPMGH'
```
Comment (Haroenv) [inline kubernetes.yml]: this should probably no the hardcoded right?

**Comment by bodinsamuel** [comment]
It launches `yarn start` too, it's just that it's bigger, managed differently and cheaper.

**Comment by Haroenv** [comment]
makes sense, it doesn't really seem much more complicated, and if it's faster & cheaper then it's good :)

**Comment by bodinsamuel** [comment]
No longer needed

**Pull request closed without merging**